digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -31,6 +34,12 @@
 public class TDOperator
         implements Closeable
 {
+    private static final Logger logger = LoggerFactory.getLogger(TDOperator.class);
+
+    private static final int INITIAL_RETRY_WAIT = 100;
```

Comment:
[minor] Yeah, but I have a little concern that this short retry wait might make the api server busier when td-api is already busy and unstable.